pkg/kubelet/nodeconfig/controller.go

```diff
@@ -0,0 +1,270 @@
+/*
```

Comment:
I am using glog, just a little deeper so I can provide context in my log messages. It's a very lightweight shim that prevents bloating the filenames, and makes it easy to search for all logs related to the controller by grepping for `nodeconfig controller`.